cumulusci/core/dependencies/dependencies.py

```diff
@@ -248,7 +248,7 @@ def _flatten_unpackaged(
             contents = None
 
         if contents:
-            for dirname in list(contents.keys()):
+            for dirname in sorted(list(contents.keys())):
```

Comment:
```suggestion
            for dirname in sorted(contents.keys()):
```

Reply:
(`.keys()` is also unnecessary, since dicts yield their keys when iterated, but helpful for clarity I think)